### PR TITLE
Added form live data events and fixed the module methods validity check

### PR DESCRIPTION
**Pull request by gabipetrovay**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,10 +21,21 @@
...
+    "name": "jillix",
+    "email": "contact@jillix.com"
+  },
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/jillix/engine-form/issues"
   },
-  "homepage": "https://github.com/jillix/engine-form"
+  "homepage": "https://github.com/jillix/engine-form",
+  "gitHead": "c26295b9bb7b4d9134a763b118addf08dfd8563a",
```
Comment (IonicaBizau) [inline package.json]: Can you please clean this up? You probably edited the npm module and then overriden everything in a clone of this repository.

File: package.json [hunk 2]
```diff
@@ -21,10 +21,21 @@
...
+  "readme": "# engine-form\nForm module for Engine.\n\n## Documentation\n### `fill(ev, data)`\nFills a form with data provided in `data.data`.\n\n#### Params\n- **Event** `ev`: The event object.\n- **Object** `data`: The data object.\n\n### `submit(ev, data)`\nRequest a submit event from form serializer.\n\n#### Params\n- **Event** `ev`: The event object.\n- **Object** `data`: The data object.\n\n## How to contribute\n1. File an issue in the repository, using the bug tracker, describing the\n   contribution you'd like to make. This will help us to get you started on the\n   right foot.\n2. Fork the project in your account and create a new branch:\n   `your-great-feature`.\n3. Commit your changes in that branch.\n4. Open a pull request, and reference the initial issue in the pull request\n   message.\n\n## License\nSee the [LICENSE](./LICENSE) file.\n",
+  "readmeFilename": "README.md",
+  "_id": "engine-form@1.0.0",
```
Comment (gabipetrovay) [inline package.json]: @IonicaBizau I assume also this?

File: package.json [hunk 3]
```diff
@@ -21,10 +21,21 @@
...
+  "_shasum": "561b92a69dd29625173a5cef1c03ae38038f13a4",
```
Comment (gabipetrovay) [inline package.json]: @IonicaBizau and all with `_`?

File: package.json [hunk 4]
```diff
@@ -21,10 +21,15 @@
...
+    "email": "contact@jillix.com"
+  },
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/jillix/engine-form/issues"
   },
-  "homepage": "https://github.com/jillix/engine-form"
+  "homepage": "https://github.com/jillix/engine-form",
+  "readme": "# engine-form\nForm module for Engine.\n\n## Documentation\n### `fill(ev, data)`\nFills a form with data provided in `data.data`.\n\n#### Params\n- **Event** `ev`: The event object.\n- **Object** `data`: The data object.\n\n### `submit(ev, data)`\nRequest a submit event from form serializer.\n\n#### Params\n- **Event** `ev`: The event object.\n- **Object** `data`: The data object.\n\n## How to contribute\n1. File an issue in the repository, using the bug tracker, describing the\n   contribution you'd like to make. This will help us to get you started on the\n   right foot.\n2. Fork the project in your account and create a new branch:\n   `your-great-feature`.\n3. Commit your changes in that branch.\n4. Open a pull request, and reference the initial issue in the pull request\n   message.\n\n## License\nSee the [LICENSE](./LICENSE) file.\n",
+  "readmeFilename": "README.md"
```
Comment (IonicaBizau) [inline package.json]: These too (`readmeFilename` and `readme`).

**Comment by gabipetrovay** [comment]
@IonicaBizau Anything else?

**Comment by gabipetrovay** [comment]
@IonicaBizau Also please check that [if condition](https://github.com/jillix/engine-form/commit/047731154c6683fcef9e21c850b21c8ef84331c9#diff-895656aeaccff5d7c0f56a113ede9662L47). I assume it was not correct.

**Comment by IonicaBizau** [comment]
@gabipetrovay Looks good! :sparkle:

**Comment by gabipetrovay** [comment]
@IonicaBizau publishing to npm? Or should we test it first from GH?

**Comment by IonicaBizau** [comment]
@gabipetrovay I'd say to use it from GitHub (master branch): `jillix/engine-form` so we will get the most updated version when `npm install`ing.

**Comment by gabipetrovay** [comment]
k

**Pull request merged**